### PR TITLE
Profile view

### PR DESCRIPTION
**Pull request by belchercd**

### CODE REVIEW

File: fbuapp/AppDelegate.m [hunk 1]
```diff
@@ -29,31 +29,36 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
+        logViewController.modalTransitionStyle = UIModalTransitionStyleCoverVertical;
+
 
-        configuration.applicationId = @"skillAppId";
-        configuration.server = @"https://skill--app.herokuapp.com/parse";
-    }];
-    [Parse initializeWithConfiguration:config];
+        UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:logViewController];
+        self.window.rootViewController = navigationController;
+    }
```
Comment (eazheng) [inline fbuapp/AppDelegate.m]:
```suggestion
    self.window.rootViewController = navigationController;
```

File: fbuapp/AppDelegate.m [hunk 2]
```diff
@@ -29,31 +29,36 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
+        logViewController.modalPresentationStyle = UIModalPresentationFullScreen;
+        logViewController.modalTransitionStyle = UIModalTransitionStyleCoverVertical;
+
 
-        configuration.applicationId = @"skillAppId";
-        configuration.server = @"https://skill--app.herokuapp.com/parse";
-    }];
-    [Parse initializeWithConfiguration:config];
+        UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:logViewController];
+        self.window.rootViewController = navigationController;
```
Comment (eazheng) [inline fbuapp/AppDelegate.m]:
```suggestion

```

File: fbuapp/AppDelegate.m [hunk 3]
```diff
@@ -29,31 +29,36 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
-        UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController: registerViewController];
+    //create for persisting user
+    if ([PFUser currentUser] != nil) {
+        //if user is already logged in, take them to HomeViewController
+        HomeViewController *homeViewController = [[HomeViewController alloc] init];
+        homeViewController.modalPresentationStyle = UIModalPresentationFullScreen;
+        homeViewController.modalTransitionStyle = UIModalTransitionStyleCoverVertical;
+
+        UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:homeViewController];
         self.window.rootViewController = navigationController;
```
Comment (eazheng) [inline fbuapp/AppDelegate.m]:
```suggestion
       
```

File: fbuapp/AppDelegate.m (hunk 1 above)
Comment (eazheng) [reply fbuapp/AppDelegate.m]: move to outside of if else statement

File: fbuapp/View Controllers/ProfileViewController.m [hunk 4]
```diff
@@ -23,18 +23,31 @@ @implementation ProfileViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    
+    //Set current user's information to their profile page.
+    PFUser *currentUser = [PFUser currentUser];
+    if (currentUser != nil) {
```
Comment (eazheng) [inline fbuapp/View Controllers/ProfileViewController.m:42]: maybe add an "else" with NSLog to show that there's an error/shouldn't have reached profileviewcontroller if PFuser is nil

File: fbuapp/View Controllers/RegisterViewController.m [hunk 5]
```diff
@@ -9,96 +9,106 @@
...
                 //get user's first and last name
-                //setting my created properties to currentProfile properties
                 self.firstNameField.text = profile.firstName;
                 self.lastNameField.text = profile.lastName;
-                
                 //test
                 NSLog(@"Hello %@!", profile.firstName);
                 
-                //welcome user to app in navigation bar
+                //Welcome user
```
Comment (danthecodingman) [inline fbuapp/View Controllers/RegisterViewController.m]: Not sure if these comments are necessary.

File: fbuapp/View Controllers/RegisterViewController.m [hunk 6]
```diff
@@ -9,96 +9,106 @@
...
+    self.signupButton.layer.shadowOpacity = 1.0f;
+    self.signupButton.layer.shadowRadius = 0.0f;
+    self.signupButton.layer.cornerRadius = 4.0f;
+    
+    //make profile image circular
+    self.profilePictureView.layer.cornerRadius = self.profilePictureView.frame.size.width / 2;
+    self.profilePictureView.clipsToBounds = YES;
+    //add border to profile image
+    self.profilePictureView.layer.borderWidth = 3.0f;
+    self.profilePictureView.layer.borderColor = [UIColor whiteColor].CGColor;
```
Comment (danthecodingman) [inline fbuapp/View Controllers/RegisterViewController.m:70]: This code is very similar to that in ProfileViewController. Perhaps make some helper methods to be used in both.

**Review by danthecodingman — COMMENTED** [review]
Method abstraction

**Pull request merged**